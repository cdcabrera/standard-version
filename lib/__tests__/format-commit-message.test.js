const formatCommitMessage = require('../format-commit-message');

describe('formatCommitMessage', () => {
  it('should format a commit message', () => {
    expect(formatCommitMessage('lorem ipsum, {{currentTag}}', '0.0.0')).toMatchSnapshot('format');
  });
});
