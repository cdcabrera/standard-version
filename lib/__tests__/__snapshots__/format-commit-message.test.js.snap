// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatCommitMessage should format a commit message: format 1`] = `"lorem ipsum, 0.0.0"`;
